### PR TITLE
fix #35454 flashplayer.ru

### LINKED ISSUE #35454: flashplayer.ru
### Issue URL (Ads)
[http://www.flashplayer.ru/tanks/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.flashplayer.ru%2Ftanks%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.1.3
Browser: | Chrome
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>EasyList,<br>AdGuard Mobile Ads,<br>RU AdList,<br>AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>Peter Lowe's list,<br>RU AdList: Counters,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/35454

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -90,6 +90,9 @@ rb.ru##.read_also
 ||wrzgu.com^$domain=lenkino.net
 hardwareluxx.ru###rec1
 hardwareluxx.ru###rec2
+||flashplayer.ru/stopvideo*.php
+flashplayer.ru###quiz
+flashplayer.ru##.gametiles > div[id]
```
Comment (BlazDT) [inline RussianFilter/sections/specific.txt]: When you hide this element, you should modify the three elements with "style="width: 181px; margin: 0px 0px 379px;"" too, else user has white space.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: Leftover
image
But i am not sure if it is ad. Better block in Annoyances.

Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: And ad leftover:
`flashplayer.ru##.main > div[id][style*="min-height"][style*="position"][style*="overflow"]`

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -90,6 +90,11 @@ rb.ru##.read_also
 ||wrzgu.com^$domain=lenkino.net
 hardwareluxx.ru###rec1
 hardwareluxx.ru###rec2
+||flashplayer.ru/stopvideo*.php
+flashplayer.ru###quiz
+flashplayer.ru##.gametiles > :not(li)
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: dangerous rule. That element has dimensions and another attributes

File: RussianFilter/sections/specific.txt [hunk 3]
```diff
@@ -90,6 +90,11 @@ rb.ru##.read_also
...
+flashplayer.ru##.playgame > :not(#gameframe)
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: also dangerous. Always add at least tags of element with `:not()`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**